system_tests/fast_confirm_test.go

```diff
@@ -40,9 +44,109 @@ import (
 	"github.com/offchainlabs/nitro/validator/valnode"
 )
 
+func TestFastConfirmationWithdrawal(t *testing.T) {
```

Comment:
It can be run in parallel?

```suggestion
func TestFastConfirmationWithdrawal(t *testing.T) {
	t.Parallel()
```